internal/commands/user/delete.go

```diff
@@ -60,11 +55,18 @@ func (cmd *CommandDelete) Handler(profile *cli.Profile, ui terminal.UI) error {
 	if appErr != nil {
 		return appErr
 	}
-	users, usersErr := cmd.inputs.resolveUsers(ui, cmd.realmClient, app)
-	if usersErr != nil {
-		return usersErr
+
+	resolvedUsers, resolveErr := cmd.inputs.resolveUsers(cmd.realmClient, app.GroupID, app.ID)
+	if resolveErr != nil {
+		return resolveErr
+	}
+
+	selectedUsers, selectErr := cmd.inputs.selectUsers(ui, resolvedUsers, "delete")
+	if selectErr != nil {
+		return selectErr
```

Comment:
[nit] keep the prefix consistent
`resolvedUsers, resolveErr` -> `resolvedUsers, resolvedErr`
`selectedUsers, selectErr ` -> `selectedUsers, selectedErr`